Makefile.build

```diff
@@ -195,7 +195,7 @@ TIMER=$(if $(TIMED), $(STDTIME), $(TIMECMD))
 COQOPTS=$(NATIVECOMPUTE)
 BOOTCOQC=$(TIMER) $(COQTOPBEST) -boot $(COQOPTS) -compile
 
-LOCALINCLUDES=$(if $(filter plugins/%,$@),-I lib -I API -open API $(addprefix -I plugins/,$(PLUGINDIRS)),$(addprefix -I ,$(SRCDIRS)))
+LOCALINCLUDES=$(addprefix -I ,$(SRCDIRS))
 MLINCLUDES=$(LOCALINCLUDES) -I $(MYCAMLP4LIB)
```

Comment:
Without API there is no need for the duplication between MLDFILE/PLUGMLDFILE and MLLIBDEFILE/PLUGMLLIBDFILE. 
This also means that the `if filter plugins/% ...` in DEPFLAGS in not useful, so that line should be `DEPFLAGS=$(LOCALINCLUDES) -I ide -I ide/utils`.
Possibly we could switch to `:=` for LOCALINCLUDES MLINCLUDES and DEPFLAGS (maybe others too).

Reply:
I'm not so sure I follow here, I don't think we want plugins to see stuff in `ide/` (eg one day we may want to move coqide into a separate repo).

Reply:
>I'm not so sure I follow here, I don't think we want plugins to see stuff in ide/ (eg one day we may want to move coqide into a separate repo).

What plugins see is determined by MLINCLUDES, having separate dependency computing just means you get warnings before the compilation error. I don't think that's worth having duplicated rules.
But I won't oppose merging for this.